spec/services/frameworks/section_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Frameworks::Section do
+  describe '#call' do
+    it 'returns a list of question keys mapped to questions' do
+      filepath = Rails.root.join(
+        'spec/fixtures/files/frameworks/person-escort-record-1/manifests/health-information.yml',
```

Comment:
I think there's a path helper for fixture files... can't recall the exact name off the top of my head though.

Reply:
It's `fixture_path` but wouldn't help much here as its defined to `../spec/fixtures/`  https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/3d06c27f5cb6be777c8f96c511612161ed201e04/spec/rails_helper.rb#L55
That's why I went with this way instead. I tidied it up a bit more though so let me know your thoughts